test/test_html2text.py

```diff
@@ -173,6 +173,12 @@ def test_func(self):
     if base_fn not in ['bodywidth_newline.html', 'abbr_tag.html']:
         test_func = None
 
+    if base_fn == 'inplace_baseurl_substitution.html':
+        module_args['baseurl'] = 'http://brettterpstra.com'
+        module_args['body_width'] = 0
+        # there is no way to specify baseurl in cli :(
```

Comment:
It depends on whether people use cli with saved html pages or not.
I use cli occasionally for testing purposes with http://* addresses only. So personally I don't need such feature.